CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ Bugfixes:
   - prioritise explicitly requested gems in dependency resolution sort order (@segiddins)
   - reduce memory usage during dependency resolution ([#6114](https://github.com/bundler/bundler/issues/6114), @greysteil)
   - ensure that the default bundler gem is not accidentally activated on ruby 2.5 when using local git overrides (@segiddins)
+  - fix `NoMethodError` when using the `--group` option with `bundle update` ([#6156](https://github.com/bundler/bundler/issues/6156), @mattbrictson)
```

Comment:
Please don't amend the Changelog, changes are added when we make a new release

Reply:
Sorry! I'll remove it. I was following the [instructions](https://github.com/bundler/bundler/blob/master/doc/development/PULL_REQUESTS.md#changelogmd) in Bundler's documentation for submitting pull requests:

> Don't forget to add your changes into the CHANGELOG! If you're submitting documentation, note the changes under the "Documentation" heading.

Reply:
@mattbrictson thanks for catching this! we automated the changelog since those docs were written. We'll update it. :)